roles/validate/files/rules/required_rules/502_policy_vrf_lite_cross_reference.py

```diff
@@ -0,0 +1,257 @@
+'''
+Validation Rules scenarios:
+1.  Switches in the [insert vrf-lite switch model key structure here] should be defined in the vxlan.topology.switches
```

Comment:
Please replace [insert vrf-lite switch model key structure here] with the actual model structure

Reply:
rewrite to `1.  Switches in the vxlan.overlay_extensions.vrf_lites.switches should be defined in the vxlan.topology.switches`